packages/back-end/src/routers/organizations/organizations.controller.ts

```diff
@@ -689,11 +689,23 @@ export async function getOrganization(
   if (licenseKey || process.env.LICENSE_KEY) {
     // automatically set the license data based on org license key
     license = getLicense(licenseKey || process.env.LICENSE_KEY);
-    if (!license || (license.organizationId && license.organizationId !== id)) {
+    if (
+      !license ||
+      (license.organizationId && license.organizationId !== id) ||
+      (license && !license.stripeSubscription) ||
+      (license && !license.orbSubscription)
+    ) {
       try {
         license =
-          (await licenseInit(org, getUserCodesForOrg, getLicenseMetaData)) ||
-          null;
+          (await licenseInit(
+            org,
+            getUserCodesForOrg,
+            getLicenseMetaData,
+            // if license is set, but no stripe or orb subscription, force refresh
```

Comment:
I think this could just be `!!license // if the license is set but in not a correct state, force refresh`.